src/rustdoc-json-types/lib.rs

```diff
@@ -233,6 +234,7 @@ pub enum ItemEnum {
         default: Option<String>,
     },
     AssocType {
+        generics: Generics,
```

Comment:
Their should be tests for json `AssocType` with `generics`, and also one with no `generics`